frontend/src/metabase/nav/components/AppBar/AppBar.tsx

```diff
@@ -16,16 +16,22 @@ export interface AppBarProps {
   isProfileLinkVisible?: boolean;
   isCollectionPathVisible?: boolean;
   isQuestionLineageVisible?: boolean;
+  isVisible: boolean;
   onToggleNavbar: () => void;
   onCloseNavbar: () => void;
   onLogout: () => void;
 }
 
-const AppBar = (props: AppBarProps): JSX.Element => {
+const AppBar = (props: AppBarProps): JSX.Element | null => {
```

Comment:
Should we just use `ReactNode` type here?

Reply:
apparently not. I can just delete this and let it infer the return type.

![Screen Shot 2023-03-16 at 3 12 48 PM](https://user-images.githubusercontent.com/30528226/225753983-2d4543a5-87db-416d-a62e-be7e10c42977.png)
![Screen Shot 2023-03-16 at 3 12 33 PM](https://user-images.githubusercontent.com/30528226/225753985-75822b38-16d4-4e68-bb2a-e20a2002d584.png)